.github/workflows/test.yml

```diff
@@ -54,3 +54,29 @@ jobs:
         env:
           NODE_OPTIONS: --max_old_space_size=4096
       - uses: codecov/codecov-action@v2
+
+  slither:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 12.x
+      - run: yarn install
+      - run: yarn compile
```

Comment:
This project doesn't use Yarn. Installing dependencies should be done with npm like the other jobs in this file.